### DISCUSSION TITLE
Poste aqui seu e-mail para ser convidado ao Slack

### DISCUSSION THREAD

**Discussion by igorlvicente** [DyegoCosta/trabalhando-remoto#128] [General]
Tente entrar pelo link:
https://join.slack.com/t/trabalho-remoto/shared_invite/zt-13v1djez9-CbazpS49PELSJKAWkw9IEw (Postado em 01/03/2021. Funcionará por 19 dias)

**Caso não consiga**, poste aqui seu e-mail para que outros possam te convidar.

**Comment by galmeida125** [comment]
gronniea@gmail.com

**Reply by igorlvicente** [reply]
Adicionado.

**Comment by fredzica** [comment]
fredzica@gmail.com

**Reply by igorlvicente** [reply]
Alguém já adicionou.

**Reply by fredzica** [reply]
valeu! eu tava lá há uns anos já, nem lembrava mais

**Comment by flplima** [comment]
felipe.lds@live.com

**Comment by igorlvicente** [comment]
Como não estou online 24h e o @flplima teve que esperar 18 horas pra ser convidado, também vou colocar o link aqui.
Quando o link deixar de funcionar, aí sim dá pra colocar e-mails de novo.
Isso vai fazer com que eu (ou outros) saibam que o link não funciona mais e geraremos outros.

https://join.slack.com/t/trabalho-remoto/shared_invite/zt-ocomur3b-5gNAzjdQJkBaesXaH4CLXQ

**Comment by ericcoutinho** [comment]
***@***.***

Em seg, 22 de mar de 2021 04:59, Igor L. Vicente ***@***.***>
escreveu:

**Reply by igorlvicente** [reply]
Não saiu seu e-mail na mensagem.

**Comment by GabrielFerrarini** [comment]
gabriel.ferrarini@gmail.com

**Comment by felipythondev** [comment]
felipejpa15@gmail.com

**Comment by borgateo** [comment]
borgato.matteo@gmail.com

Obg!

**Comment by afmatheus** [comment]
matheus.fonseca.alves@gmail.com

**Comment by lmcarreiro** [comment]
leonardo.mcarreiro@gmail.com

**Comment by EltonGarcia** [comment]
elton.garcia.s@gmail.com

**Comment by andersoonluan** [comment]
andersoonluan@gmail.com

**Comment by clarissalimab** [comment]
clarissalimab08@gmail.com

**Comment by mathbteixeira** [comment]
mbateixeira@yahoo.com.br

**Comment by renanfferreira** [comment]
renanfelipeferreira@live.com

**Comment by pedropaulofreitas** [comment]
pedrofreitas@poli.ufrj.br

**Comment by rodrigues882013** [comment]
rodrigues882007@gmail.com

**Comment by spadilha** [comment]
spadilha@gmail.com

**Comment by cesdias** [comment]
cesdias@protonmail.com

**Comment by jkeilla** [comment]
jkeilla017@gmail.com

**Comment by kedson42** [comment]
kedson.42@gmail.com

**Comment by alisonjf** [comment]
alisonjferrenha@gmail.com

**Comment by brunoklein99** [comment]
brunoklein99@gmail.com

**Comment by nythrox** [comment]
jasonsantiagobutler@gmail.com

**Comment by carloseustaquio** [comment]
caeu227@gmail.com

**Comment by thiagohsr** [comment]
thiagoramos@gmail.com

**Comment by ap-pauloafonso** [comment]
ap.pauloafonso@gmail.com

**Comment by natanaelrw** [comment]
natanaelrw@gmail.com

**Comment by lucasmro** [comment]
***@***.***

Em dom., 31 de out. de 2021 às 22:04, Marton Paulo ***@***.***>
escreveu:

> ***@***.***
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DyegoCosta/trabalhando-remoto/discussions/128#discussioncomment-1567379>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAP5YCOCMPE47DR6YJB5LVLUJXRRHANCNFSM4ZSTD63Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 

--

Lucas Michelini Reis de Oliveira
[ Gtalk ] ***@***.***
[ Skype ] lucasmro

**Reply by igorlvicente** [reply]
Não veio. Acesse pelo link.

**Comment by Joaovtrc** [comment]
jvromerocruz@gmail.com

**Comment by rafaeldbernardes** [comment]
rafaeldbernardes@gmail.com

**Comment by Giuco** [comment]
giucoferrari@gmail.com

**Comment by l4p4** [comment]
***@***.***

On Wed, Nov 3, 2021 at 6:40 PM Giuliano Ferrari ***@***.***>
wrote:

> ***@***.***
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DyegoCosta/trabalhando-remoto/discussions/128#discussioncomment-1585028>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AARYDXXZP5OUBBZ5KIHYNIDUKGT4HANCNFSM4ZSTD63Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Best regards,
Larry Pavanery

**Reply by igorlvicente** [reply]
Seu e-mail não aparece. Tente acessar pelo link.

**Comment by danielbergholz** [comment]
bergholz.daniel@gmail.com

**Comment by matusca96** [comment]
gomesloud@gmail.com

**Comment by eduardofischer** [comment]
eduardo.sfischer@gmail.com

**Comment by miguelriosoliveira** [comment]
miguelriosoliveira@gmail.com

**Comment by ap-pauloafonso** [comment]
ap.pauloafonso@gmail.com

**Comment by gustavoaraujo** [comment]
gustavobr1989@gmail.com

**Comment by lima-emanuel** [comment]
emanuel-lima@outlook.com

**Comment by andersoonluan** [comment]
andersoonluan@gmail.com

**Comment by andre-salvati** [comment]
andre.f.salvati@gmail.com

**Comment by maxmiliano** [comment]
***@***.***

Valeu, galera! 
Entrei no Slack e vou tirar o e-mail daqui.

Sei que deve ficar no histórico, mas já dificulta um pouco pro crawler, né!?

**Comment by evandrocurrencies** [comment]
lopes.evandro@gmail.com

**Comment by mrneitzke** [comment]
mrneitzke@gmail.com

**Comment by andretessmann-as** [comment]
andre.tessmann@alphasights.com

**Comment by mrneitzke** [comment]
mrneitzke@gmail.com

**Comment by lucasalmeida92** [comment]
lucasalmeida70@gmail.com

**Comment by vinicius-conceicao** [comment]
viniciusoliveira.luiz@hotmail.com

**Comment by felipe-rodrigues** [comment]
feliperodriguessilva.ti@gmail.com

**Comment by igorlvicente** [comment]
Ultimamente eu estou usando um computador da empresa e não posso entrar em outros canais do Slack. Quem estiver no canal, por favor, convide os e-mails aqui da thread e gere um novo link.

**Comment by gui-esp** [comment]
guilherme.ntdi@gmail.com

**Comment by andersoonluan** [comment]
Alguem poderia me convidar para o Grupo por favor? Meu email é: andersoonluan@gmail.com

**Comment by klebermagno** [comment]
Klebermagno at gmail.com

**Comment by rodrigo-om** [comment]
rodrigo.om96@gmail.com

**Comment by brunokim** [comment]
brunokimmc@gmail.com

**Comment by bsda** [comment]
bruno@savioli.org

**Comment by henriquetatagiba** [comment]
henriquectatagiba@gmail.com

**Comment by iacapuca** [comment]
plpbs@poli.br

**Comment by pagottoo** [comment]
pagottoo@gmail.com

**Comment by rodrigoaps** [comment]
rodrigoapdesouza@gmail.com

**Comment by rodrigolosik** [comment]
rodrigolosik@gmail.com

**Comment by brnpimentel** [comment]
bruno@brunopimentel.dev

**Comment by denisdifazio** [comment]
denisdifazio@gmail.com

**Comment by caiquefq** [comment]
caique.f.queiroz@gmail.com

**Comment by marcosfsch** [comment]
marcos.f.sch@gmail.com

**Comment by danferreira** [comment]
danferreiradev@gmail.com

**Comment by jdanieel** [comment]
jdanieel@gmail.com

**Comment by andrerip** [comment]
andrerip@gmail.com

**Comment by brnpimentel** [comment]
bruno@brunopimentel.dev

**Comment by jocieldo** [comment]
jocieldo@live.com

**Comment by alandefreitas** [comment]
alandefreitas@gmail.com

**Comment by eduardogranetto** [comment]
eduardogranetto@gmail.com

**Comment by AndersonAltissimo** [comment]
ander.altissimo@gmail.com

**Comment by alandefreitas** [comment]
Está rolando ainda?

**Comment by victorrumanski** [comment]
victor.rumanski@gmail.com

**Comment by cleversonledur** [comment]
cleversonledur@gmail.com

**Comment by lucastrvsn** [comment]
lucastrvsn@gmail.com

**Reply by hugows** [reply]
entrou ?

**Comment by Vrossi28** [comment]
vrossi28@outlook.com

**Comment by hugows** [comment]
hugows@gmail.com

**Comment by alandefreitas** [comment]
Me parece que os comentários aqui estão sendo ignorados. Talvez alguém aqui queira abrir outro grupo?

**Reply by viniciusdeliz** [reply]
Ao invés de um slack, não seria mais conveniente fazer um canal/grupo no Discord?

**Reply by alandefreitas** [reply]
Acho que tanto faz. No trabalho eu já uso slack então já fica aberto, mas acho que cada um usa um então tanto faz. Provavelmente o que a maioria usa é melhor.

**Comment by viniciusdeliz** [comment]
vinicius.deliz@gmail.com

**Comment by emwp** [comment]
everton.mwp@gmail.com

**Comment by ramoncaldeira** [comment]
ramon.caldf@gmail.com

**Comment by rCirelli** [comment]
raphael.cirelli@protonmail.com

**Comment by andreksu** [comment]
deh.snk@gmail.com

**Comment by thjsantos90poe** [comment]
tjustinosantos@gmail.com

Obrigado

**Comment by hugows** [comment]
Seguindo a dica de alguém acima, abri um novo workspace: [SLACK](https://join.slack.com/t/trabalhandoremoto/shared_invite/zt-1o3xf2gnp-EB~7Zcakpi0kJesivMas3w) 🚀🚀🚀

Tomei a liberdade de convidar **todos** emails acima.

**Comment by mobtavares** [comment]
mobt.ap@gmail.com

**Comment by jozimarback** [comment]
jozimar5@gmail.com

**Comment by ribeiroevandro** [comment]
ribeiroevandro@live.com

**Comment by Tpessia** [comment]
thiago.pessia@gmail.com

**Comment by Vittalev** [comment]


**Reply by hugows** [reply]
https://join.slack.com/t/trabalhandoremoto/shared_invite/zt-1tkrmd3zo-KLrCR1~CKyS~VRelc0ElVQ

**Comment by hugows** [comment]
O jeito mais fácil de entrar é usar o link:

https://join.slack.com/t/trabalhandoremoto/shared_invite/zt-1tkrmd3zo-KLrCR1~CKyS~VRelc0ElVQ

**Comment by DanielWyllyan1** [comment]
> Tente entrar pelo link: https://join.slack.com/t/trabalho-remoto/shared_invite/zt-13v1djez9-CbazpS49PELSJKAWkw9IEw (Postado em 01/03/2021. Funcionará por 19 dias)
> 
> **Caso não consiga**, poste aqui seu e-mail para que outros possam te convidar.
williandaniel789@gmail.com

**Comment by vonkjerome** [comment]
vonkjerome@gmail.com

**Comment by lucascoelh0** [comment]
lcsilva2099@gmail.com

**Comment by Philippe-SS** [comment]
lipe.philippe64@gmail.com

**Comment by DanielWyllyan1** [comment]
dwillians@pelotongroup.com

**Comment by oordnave** [comment]
oordnave@gmail.com

**Comment by kevinmmartins** [comment]
kevinmmartins@gmail.com

**Comment by nelsonthedev** [comment]
leonardosn9@gmail.com

**Comment by avizmarlon** [comment]
aviz.marlon@gmail.com

**Comment by raphaellins** [comment]
raphalinns@gmail.com

**Comment by dougdecarli** [comment]
douglasdecarlli@hotmail.com

**Comment by RyanFilho** [comment]
> Tente entrar pelo link: https://join.slack.com/t/trabalho-remoto/shared_invite/zt-13v1djez9-CbazpS49PELSJKAWkw9IEw (Postado em 01/03/2021. Funcionará por 19 dias)
> 
> **Caso não consiga**, poste aqui seu e-mail para que outros possam te convidar.

ryansampaiofilho@outlook.com

**Comment by phasath** [comment]
> Tente entrar pelo link: https://join.slack.com/t/trabalho-remoto/shared_invite/zt-13v1djez9-CbazpS49PELSJKAWkw9IEw (Postado em 01/03/2021. Funcionará por 19 dias)
> 
> **Caso não consiga**, poste aqui seu e-mail para que outros possam te convidar.

sathler93@gmail.com

**Comment by oordnave** [comment]
***@***.***

Em qua., 20 de dez. de 2023 às 15:29, Raphael Sathler <
***@***.***> escreveu:

**Comment by Bruno-Jander** [comment]
bruno-jander@hotmail.com

**Reply by Bruno-Jander** [reply]
Alguém consegue me  incluir no time aqui? :')

**Comment by mcferdev** [comment]
Se ainda utilizarem: marcelocfernandes@yahoo.com.br

**Comment by hugows** [comment]
Invite: https://join.slack.com/t/trabalhandoremoto/shared_invite/zt-1tkrmd3zo-KLrCR1~CKyS~VRelc0ElVQ

**Comment by hugows** [comment]
Pessoal eu acabei criando esta comunidade "nova" mas não tem nada por lá, pouca atividade. Caso tenham ideias, me avisem.
